src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -172,9 +196,12 @@ public void setSequenceNumberExpression(Expression sequenceNumberExpression) {
 	/**
 	 * Specify a {@link OutboundMessageMapper} for embedding message headers into the
 	 * record data together with payload.
+	 *
 	 * @param embeddedHeadersMapper the {@link OutboundMessageMapper} to embed headers
 	 * into the record data.
+	 *
 	 * @since 2.0
+	 *
```

Comment:
Please, revert all the changes in Javadocs. 
We prefer to not have blank lines in method Javadocs.